### PR TITLE
Add outputConfiguration default  - (Task #11)

### PR DESCRIPTION
**Pull request by TjorveKujathDLR**
Set same parameters as in conceptOutput.

---
Task #11: Execution of OverTarget in Maven/Tycho gives warning

### CODE REVIEW

File: de.dlr.sc.overtarget.language/src/de/dlr/sc/overtarget/language/generator/OvertargetOutputConfigurationProvider.java [hunk 1]
```diff
@@ -44,6 +49,17 @@
 		
 		outputConfigurations.add(conceptOutput);
```
Comment (SaMuellerDLR) [inline de.dlr.sc.overtarget.language/src/de/dlr/sc/overtarget/language/generator/OvertargetOutputConfigurationProvider.java]: I just noticed that the regular output configuration is called conceptOutput.
This was probably copied over from VirSat. We should change the name to something with overtarget
to make it clear what this configuraiton is.

Comment (TjorveKujathDLR) [reply de.dlr.sc.overtarget.language/src/de/dlr/sc/overtarget/language/generator/OvertargetOutputConfigurationProvider.java]: Fixed.

**Comment by TjorveKujathDLR** [comment]
@SaMuellerDLR  refactored the testcase. Please check.

File: de.dlr.sc.overtarget.language.tests/src/de/dlr/sc/overtarget/language/tests/OvertargetOutputConfigurationProviderTest.java [hunk 2]
```diff
@@ -12,59 +12,50 @@
...
-		final String GENERATOR_OUTPUT_DESCRIPTION_CONCEPT = "Xtend Overtarget Serialzation Output";
-
-		final String GENERATOR_OUTPUT_ID_DEFAULT = IFileSystemAccess.DEFAULT_OUTPUT;
-		final String GENERATOR_OUTPUT_FOLDER_DEFAULT = "./default-gen";
-		final String GENERATOR_OUTPUT_DESCRIPTION_DEFAULT = "Xtend Default Output";
-		
 		OvertargetOutputConfigurationProvider oocp = new OvertargetOutputConfigurationProvider();
 		Set<OutputConfiguration> outputConfigurations = oocp.getOutputConfigurations();
 		
-		assertEquals("The number of Output Configurations is correct", NUMBER_OF_OUTPUTS, outputConfigurations.size());
```
Comment (SaMuellerDLR) [inline de.dlr.sc.overtarget.language.tests/src/de/dlr/sc/overtarget/language/tests/OvertargetOutputConfigurationProviderTest.java]: The constant NUMBER_OF_OUTPUTS I would have left in

Comment (TjorveKujathDLR) [reply de.dlr.sc.overtarget.language.tests/src/de/dlr/sc/overtarget/language/tests/OvertargetOutputConfigurationProviderTest.java]: Fixed

Comment (TjorveKujathDLR) [reply de.dlr.sc.overtarget.language.tests/src/de/dlr/sc/overtarget/language/tests/OvertargetOutputConfigurationProviderTest.java]: @SaMuellerDLR Fix the change to NUMBER_OF_OUTPUTS. Ready to check.

**Pull request merged**